lib/main_common.pm

```diff
@@ -865,10 +865,10 @@ sub load_inst_tests {
                 loadtest "installation/partitioning_raid";
             }
             elsif (check_var('LVM', 0) && get_var('ENCRYPT')) {
-                loadtest 'installation/partitioning_crypt_no_lvm';
+                loadtest 'installation/partitioning_encrypted_no_lvm';
             }
             elsif (get_var("LVM")) {
-                loadtest "installation/partitioning_lvm";
+                loadtest "installation/partitioning_encrypted_lvm";
```

Comment:
I think you are changing the meaning here in an unintended way. This one here is probably covering lvm in both cases, unencrypted *and* encrypted